File: Desenvolvimento/server/src/ApplicationCore/Entities/AssuntoArquivo.cs
using ApplicationCore.Interfaces.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationCore.Entities
{
    public class AssuntoArquivo : IBaseEntity<long>
    {
        public long Id { get; set; }
        public long AssuntoId { get; set; }
        public virtual Assunto Assunto { get; set; }
        public long ArquivoId { get; set; }
        public virtual Arquivo Arquivo { get; set; }
    }
}

File: Desenvolvimento/client/src/@fuse/types/models/assunto-arquivo.ts
import { Assunto } from './assunto';
import { BaseModel } from '../base-model';
import { Arquivo } from './arquivo';
export class AssuntoArquivo extends BaseModel {
    constructor(
        public id: number,
        public assuntoId?: number,
        public assunto?: Assunto,
        public arquivoId?: number,
        public arquivo?: Arquivo,
    ) { super(); }
}
